public/static/locales/de/common.json

```diff
@@ -1,4 +1,5 @@
 {
+  "intro_already_booked": "You have already booked an introductory meeting with this instructor. Please continue your booking with {{instructorName}} <a class='font-bold underline' href='/{{slug}}'>here</a>.",
```

Comment:
Do we have to add strings if there isn't a translation? I would think it would show English as a fallback if the translation didn't exist, but I haven't messed with it on the cal.com stuff

Reply:
I'm not sure how it works either, in past additions i think i've just added empty strings to every translation instead of the english translation.. I could switch to that? or you're right, it may not fallback to english if the translations exist as empty strings.. 🤷

Reply:
Check to see if all english keys from the base repo are present in the foreign languages, if they aren't, then we know it isn't required to add them as it will fallback to English. I'm assuming that's how it works as that's how other i18n systems I've used before work. We're fine to add them for this PR though